test/spaces_reader/fetcher-spec.ts

```diff
@@ -205,6 +204,7 @@ describe('spaces_reader fetcher', () => {
                     });
 
                 await harness.initialize();
+                // FIXME: is msg ... a slice?
```

Comment:
Oh I meant to delete this, but it brings up a good point.  I was confused that `msg` is mostly a `slice`, would it be OK if I renamed it in all the cases where `msg=slice`?